formats/email_alert_signup/publisher/details.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "tags"
+  ],
+  "properties": {
+     "tags": {
+       "type": "object"
+    },
+    "breadcrumbs": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "link",
+          "title"
+        ],
+        "properties": {
+          "link": {
+            "type": "string",
+            "format": "uri"
+          },
+          "title": {
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9-]+$"
```

Comment:
OK - I was referring to:
```
  The property '#/details/breadcrumbs/0/title' value "An extra bank holiday" did not match the regex '^[a-zA-Z0-9-]+$' in schema file:///home/jenkins/workspace/govuk_content_schemas_branches/formats/email_alert_signup/frontend/schema.json#
```
https://ci-new.alphagov.co.uk/job/govuk_content_schemas_branches/109/console

But it's all fixed now, so no worries.